Example/Tests/BONUtilitiesTestCase.m

```diff
@@ -0,0 +1,252 @@
+//
+//  BONUtilitiesTestCase.m
+//  BonMot
+//
+//  Created by Nora Trapp on 3/3/16.
+//
+//
+
+#import "BONBaseTestCase.h"
+
+@import BonMot;
+
+@interface BONUtilitiesTestCase : BONBaseTestCase
+
+@end
+
+@implementation BONUtilitiesTestCase
+
+#pragma mark - UILabel
+
+- (void)testLabelTextAfterChainable
+{
+    UILabel *label = UILabel.new;
+    label.bonChainable = BONChain.new.font([UIFont systemFontOfSize:16]);
+    label.textAndApplyChainable = @"Hello, world!";
```

Comment:
#nth it's really weird to me to use this with dot syntax, given how the wording works out. It makes much more sense as `[label setTextAndApplyChainable:@"Hello, world!"];`. But I also like dot syntax, and since this also works in Swift, we may want to reword to optimize for dot syntax.